docs/reference/data-streams/downsampling-ilm.asciidoc

```diff
@@ -0,0 +1,548 @@
+                                                },
+                                                "limit": {
+                                                    "properties": {
+                                                        "pct": {
+                                                            "type": "float"
+                                                        }
+                                                    }
+                                                },
+                                                "node": {
+                                                    "properties": {
+                                                        "pct": {
+                                                            "type": "float"
+                                                        }
+                                                    }
+                                                }
+                                            }
+                                        },
+                                        "workingset": {
+                                            "properties": {
+                                                "bytes": {
+                                                    "type": "long",
+                                                    "time_series_metric": "gauge"
+                                                }
+                                            }
+                                        }
+                                    }
+                                },
+                                "name": {
+                                    "type": "keyword"
+                                },
+                                "start_time": {
+                                    "type": "date"
+                                }
+                            }
+                        },
+                        "host": {
+                            "type": "keyword",
+                            "time_series_dimension": true
+                        },
+                        "namespace": {
+                            "type": "keyword",
+                            "time_series_dimension": true
+                        },
+                        "node": {
+                            "type": "keyword",
+                            "time_series_dimension": true
+                        },
+                        "pod": {
+                            "type": "keyword",
+                            "time_series_dimension": true
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+----
+// TEST[continued]
+
+////
+[source,console]
+----
+DELETE _index_template/*
+----
+// TEST[continued]
+////
+
+[discrete]
+[[downsampling-ilm-ingest-data]]
+==== Ingest time series data
+
+Use a bulk API request automatically create your TSDS and index a set of ten
```

Comment:
I think thi
```suggestion
Use a bulk API request to automatically create your TSDS and index a set of ten
```

Reply:
Fixed in [507419d](https://github.com/elastic/elasticsearch/pull/88571/commits/507419dcc6e4498ad50f37505d150e5de4db8a77)